README.md

```diff
@@ -207,6 +207,24 @@ Type reset to start again and test this by adding the phrase "to eat there...":
 
 # Troubleshooting
 
+* Deploy using Cloud Foundry 'cf push' gives:
+
+``FAILED
+Could not find service wcsi-conversation-service to bind to watson-conversation-slots-intro``
+
+You can still use `cf push`, but realize that it is trying to bind to the services listed in the manifest.yml
```

Comment:
Will do. I always resist this, since the final word in the sentence has period(s) in it. But I think you are right.
Just looks funny to me..